doc/cephfs/fs-volumes.rst

```diff
@@ -1046,6 +1046,120 @@ following command.
 
 .. _subvol-pinning:
 
+Controlling Subvolume Snapshot Visibility
+-----------------------------------------
+
+.. note:: This functionality is currently supported only for FUSE clients.
+          Kernel client support is planned; progress can be tracked at
+          https://tracker.ceph.com/issues/72589.
+
+Snapshots of a subvolume can be hidden from compatible clients by
+performing two actions:
+    1) Disable the subvolume's ``snapshot_visibility`` flag (default is ``true``).
+    2) Enable the client-side configuration option ``client_respect_subvolume_snapshot_visibility``
```

Comment:
Suggest explicitly setting to true vs requiring the reader to interpret

Reply:
done https://github.com/ceph/ceph/compare/357b0f673dbfddb60b7a2e8f7b40456dfeae6d79..be8c651bcfa4275bb9430b62a47faaca34ff4c19